apps/mappings/signals.py

```
@@ -187,6 +187,16 @@ def run_pre_mapping_settings_triggers(sender: type[MappingSetting], instance: Ma
                     'field_name': instance.source_field
                 })
 
+        except SageIntacctCredential.DoesNotExist:
+            logger.error(
+                'Active Sage Intacct credentials not found for workspace_id - %s',
+                workspace_id
+            )
+            raise ValidationError({
+                'message': 'Sage Intacct credentials not found in workspace',
+                'field_name': instance.source_field
+            })
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)